test/mainnet/invariants.spec.sol

```diff
@@ -21,22 +20,22 @@ contract VaultMainnetInvariantsTest is Test, AssertUtils, MainnetActors {
 
     function totalSupplyInvariant(uint256 supply) public view {
         uint256 finalVaultTotalSupply = vault.totalSupply();
-        assertEqThreshold(
-            supply, finalVaultTotalSupply, 3, "Vault totalSupply should be original totalSupply plus additional"
+        assertApproxEqRel(
+            supply, finalVaultTotalSupply, 1e14, "Vault totalSupply should be original totalSupply plus additional"
```

Comment:
why is this done what changed?

Reply:
With the addition of more assets and particularly the lp pool strategy, the variations are a little above the "3" threshold that was there earlier. I changed it to use `assertApproxEqRel` with 0.01 % threshold to have a more deterministic test